### ISSUE TITLE
the model we use

### ISSUE THREAD

**Issue by QueenJuliaZxx** [thtrieu/darkflow#433]
hi,you
1   I wonder whether  the model include in this project I download is a pre-model on imagenet or pre-trained on imagenet and then train on coco/voc,for example,the yolo_voc.cfg and yolo_voc.weight,what knd of model it belong to? 
2  Why I use the yolo_voc.weight and .cfg downloaded  trained on The Kitti dataset,the loss is stoped between 3-6,shouldnt it be very small?
